client/pausemenutitle.lua

```diff
@@ -0,0 +1,8 @@
+RegisterNetEvent('QBCore:Client:OnPlayerLoaded', function()
+	local name = GetPlayerName(PlayerId())
+	local id = GetPlayerServerId(PlayerId())
+    AddTextEntry("PM_PANE_LEAVE", "~r~Quit ~w~de ~y~QBCore ☹️")
+	Citizen.InvokeNative(GetHashKey("ADD_TEXT_ENTRY"), 'FE_THDR_GTAO', "☠️~w~~r~QBCore~b~RP~s~☠️ | ~b~Discord : ~g~https://discord.gg/qbcore~s~ | ~b~ID: "..id.." | ~r~")
+	AddTextEntry("PM_PANE_QUIT", "~r~Leave ~o~QBCore RP ☠️")
+	Wait(10000)

```

Comment:
What exactly is the purpose of this 10 second wait at the end of the function? Also why did you randomly use Citizen.InvokeNative instead of AddTextEntry like you did above and underneath it